docs/resources/port_action.md

```diff
@@ -480,17 +489,17 @@ Optional:
 
 Optional:
 
-- `boolean_items` (Attributes) The items of the array property (see [below for nested schema](#nestedatt--self_service_trigger--user_properties--array_props--boolean_items))
+- `boolean_items` (Attributes) An array of boolean items within the property (see [below for nested schema](#nestedatt--self_service_trigger--user_properties--array_props--boolean_items))
```

Comment:
aren't these files auto generated with gen-doc?
if so than these changes will be overriden next time the resources are updated and the docs re-generated

Reply:
yes, generated by `gen-doc`

Reply:
Then we should change the `MarkdownDescription` of the properties in the relevant `schema.go` files and rerun gen-doc instead of manually changing the files here

Reply:
should I leave it be ?

Reply:
NVM I see now that the schemas are changed and its not manually edited in the docs 🤦‍♂️